sentence_transformers/sparse_encoder/models/SpladePooling.py

```diff
@@ -28,14 +32,20 @@ class SpladePooling(Module):
                 - `relu`: ReLU activation (standard in all Splade models).
                 - `log1p_relu`: log(1 + ReLU(x)) variant used in Opensearch Splade models see arxiv.org/pdf/2504.14839.
         word_embedding_dimension (int, optional): Dimensionality of the output embeddings (if needed).
+        chunk_size (int, optional): Size of chunks when processing tokens. If None, processes entire sequence at once.
+            Using smaller chunks reduces memory usage but may impact speed performance. Default is None.
```

Comment:
`chunks` is not very clear, normally I would think that this chunks in the batch dimension and not in the sequence length dimension.